MapleServer2/PacketHandlers/Game/GuildHandler.cs

```diff
@@ -462,12 +462,12 @@ private void HandleUseBuff(GameSession session, PacketReader packet)
             // TODO: Send buff packet
         }
 
-        private void HandleEnterHouse(GameSession session, PacketReader packet)
+        private static void HandleEnterHouse(/*GameSession session, PacketReader packet*/)
```

Comment:
We could alternatively just delete the parameters, but this is to remove unused parameters. I left them commented for future implementation sake.

Reply:
This is actually a case where I might leave them uncommented. Yes, it throws compiler warnings, but those warnings remind you that it hasn't been implemented yet. 

Otherwise, leave them commented out for now, it's probably better for reference for the future implementation.